### ISSUE TITLE
[FEATURE] Provide the user with an option to resume the test or not.

### ISSUE THREAD

**Issue by abelparayil** [prakhartiwari0/Arito#70]
Also, I was thinking will it better if we provide the user with an option to resume the test or not.
Because if the user doesn't want to resume the test, they will be able to discard it. 

What do you think about this? @Anan7Codes 
And can you implement this feature?

_Originally posted by @prakhartiwari0 in https://github.com/prakhartiwari0/Arito/issues/53#issuecomment-1383579544_

I would like to work on this feature.

**Comment by prakhartiwari0** [comment]
Hey @abelparayil Thanks for raising this issue for this new feature! Yes you are right, and @Anan7Codes also said this thing to be solved in another issue in this reply https://github.com/prakhartiwari0/Arito/pull/53#issuecomment-1385151485

Go ahead, you are now assigned for this issue 
Happy Contributing!

**Comment by abelparayil** [comment]
Sure. I have started working on it @prakhartiwari0

**Comment by prakhartiwari0** [comment]
LOL I mistakenly closed this issue and didn't notice, sorry for that!

**Comment by Anan7Codes** [comment]
> Owner

Haha, no worries.

**Comment by abelparayil** [comment]
Nah, it's fine! Anyways, I have opened a pull request. Let me know if anymore changes have to be done!

**Comment by prakhartiwari0** [comment]
@abelparayil Looks like great work, I would need some time to go through the changes and find out any bugs or other issues. Thanks for your contribution!

Happy Contributing!
